docs/docs/legal/_piid-table.mdx

```diff
@@ -0,0 +1,93 @@
+<table>
+  <tr>
+    <th>Type of personal data</th>
+    <th>Examples</th>
+    <th>Affected data subjects</th>
+  </tr>
+  <tr>
+    <td><strong>Basic data</strong></td>
+    <td>
+      <ul>
+        <li>Surname and first name</li>
+        <li>Email addresses</li>
+        <li>User name</li>
+      </ul>
+    </td>
+    <td>All users</td>
+  </tr>
+  <tr>
+    <td><strong>Login data</strong></td>
+    <td>
+      <ul>
+        <li>Randomly generated ID</li>
+        <li>Password</li>
+        <li>Public keys / certificates ("FIDO2", "U2F", "x509", ...)</li>
+        <li>User names or identifiers of external login providers</li>
+        <li>Phone number(s)</li>
+      </ul>
+    </td>
+    <td>
+      <p>All users</p>
+      <p>Password: Users who use authentication methods with password.</p>
+      <p>Public Keys: Users who use an authentication procedure with cryptographic keys.</p>
+      <p> External login provider identifiers: Users who use an external login provider.</p>
+      <p>Phone number: Users who use authentication methods with SMS</p>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Profile data</strong></td>
+    <td>
+      <ul>
+        <li>Profile pictures</li>
+        <li>Gender</li>
+        <li>Language</li>
+        <li>Nickname</li>
+        <li>Display name</li>
+        <li>Phone number(s)</li>
+      </ul>
+    </td>
+    <td>Users who voluntarily add profile data</td>
+  </tr>
+  <tr>
+    <td><strong>Communication data</strong></td>
+    <td>
+      <ul>
+        <li>Emails</li>
+        <li>Chats</li>
+        <li>Call metadata</li>
+      </ul>
+    </td>
+    <td>Customers and users who communicate with us directly (e.g. support)</td>
+  </tr>
+  <tr>
+    <td><strong>Payment data</strong></td>
```

Comment:
On the mobile phone the user can also choose apple and google pay, are there any other infos needed?

Reply:
This is covered with payment information. This could be a link to an account number, credit card, or alternative payment methods that are processed by Stripe.